CENNZnet_format.md

```diff
@@ -47,14 +47,16 @@ The domain is binary encoded. It contains a version definition, and a permission
             * 1 bit
                 * `block_cooldown` presence
             * 7 bits
-                * Reserved/unused
+                * `constraints_count`
```

Comment:
we only need 1 bit to signal whether `constraints` is included or not i.e.
```
* 1 bit
    * `block_cooldown` presence
* 1 bit
    * `constraints` presence
* 6 bits
    * reserved/unused
```

Can decode the length byte as part of `constraints` decoding, see below